ui/app/src/components/actionsPanel/ActionsPanel.vue

```diff
@@ -22,7 +22,9 @@ export default defineComponent({
   setup(_, { emit }) {
     return {
       handleNextStepClicked() {
-        emit("nextstepclick");
+        if (_.nextStepAllowed) {
```

Comment:
This is not the best way to solve this. I think it has to do with the SifButton component. What is happening is that we are not delegating event handlers to the button and link components within SifButton.

Reply:
I think this is now addressed by removing the wrapping `span` so that click handler is attached directly to the `button` element.